jetty-core/jetty-websocket/jetty-websocket-jetty-common/src/main/java/org/eclipse/jetty/websocket/common/JettyWebSocketFrameHandlerMetadata.java

```diff
@@ -99,15 +93,15 @@ public MethodHandle getFrameHandle()
         return frameHandle;
     }
 
-    public void setOpenHandler(MethodHandle open, Object origin)
+    public void setConnectHandle(MethodHandle connectHandle, Object origin)
```

Comment:
This is a bad name.

Per https://datatracker.ietf.org/doc/html/rfc6455  the named states are ....

* CONNECTING - for when the upgrade has started, and the opening handshake is underway (no frames yet)
* OPEN - for when the upgrade handshake is complete and the two endpoints can now send/receive frames.
* CLOSING - for when the close handshake has begun (limited number of frames can still be sent/received here)
* CLOSED - for when the websocket is fully closed and no more frames can be sent/received.

Reply:
Renamed to "open".